### PR TITLE
#CART-1173 Update CVAT Scripts

### PR DESCRIPTION
**Pull request by frndmg**
* `cvat/apps/engine/management/commands/init_task.py`: importing
* `cvat/apps/engine/management/commands/export_completed_annotations.py`: exporting and deleting

### CODE REVIEW

File: cvat/apps/engine/management/commands/export_completed_annotations.py [hunk 1]
```diff
@@ -65,7 +65,7 @@ def handle(self, *args, **options):
 
 
 def dump_annotations(task, dump_folder, overwrite=False):
-    if task.assignee.username != 'bot':
+    if task.assignee is not None and task.assignee.username != 'bot':
```
Comment (MultifokalHirn) [inline cvat/apps/engine/management/commands/export_completed_annotations.py]: should have had this bevore but maybe return if assignee is None?

Comment (frndmg) [reply cvat/apps/engine/management/commands/export_completed_annotations.py]: This is a completed task. Shouldn't it be dumped? The rest of the code does not depend on the assignee anyway.

Comment (MultifokalHirn) [reply cvat/apps/engine/management/commands/export_completed_annotations.py]: I mean this is a complete edge case, but I dont think we should export something if we dont know who annotated it

**Review by MultifokalHirn — APPROVED** [review]

**Comment by MultifokalHirn** [comment]
@frndmg did you check whether all files get deleted as well?

**Comment by MultifokalHirn** [comment]
> @frndmg did you check whether all files get deleted as well?

?

File: cvat/apps/engine/management/commands/export_completed_annotations.py [hunk 2]
```diff
@@ -74,3 +74,12 @@ def dump_annotations(task, dump_folder, overwrite=False):
...
     dump_annotation_for_task(task, output_folder, overwrite=overwrite)
+
+def delete_task(task):
+    delete_task_data(task.id, task.assignee)
+
+    shutil.rmtree(task.get_task_dirname(), ignore_errors=True)
+    shutil.rmtree(task.data.get_data_dirname(), ignore_errors=True)
+
+    task.data.delete()
+    task.delete()
```
Comment (MultifokalHirn) [inline cvat/apps/engine/management/commands/export_completed_annotations.py:85]: whats the difference between task.data.delete() and delete_task_data(task.id, task.assignee)?

what about something like

```
def delete_task(task):
    try:
        delete_task_data(task.id, task.assignee)
        shutil.rmtree(task.get_task_dirname(), ignore_errors=True)
        shutil.rmtree(task.data.get_data_dirname(), ignore_errors=True)
        task.data.delete()
    except Exception as e: 
        print(f"Could not delete task data for {task}!")
        print(e)
    else:
        task.delete()
```

Comment (frndmg) [reply cvat/apps/engine/management/commands/export_completed_annotations.py:85]: @MultifokalHirn 

- `delete_task_data` is meant for removing annotations (and annotations data) related to this task, it is implemented on `cvat/apps/engine/annotation.py`.
- `task.data.delete()` will only remove this data entry from the db.

Comment (MultifokalHirn) [reply cvat/apps/engine/management/commands/export_completed_annotations.py:85]: ah ok thanks. 
the exception handling is unnecessary I guess..
I think I can merge right?

**Pull request merged**